backend/pkg/plugins/plugins.go

```diff
@@ -77,20 +77,30 @@ func periodicallyWatchSubfolders(watcher *fsnotify.Watcher, path string, interva
 	}
 }
 
-// GeneratePluginPaths takes the staticPluginDir and pluginDir and returns a list of plugin paths.
-func GeneratePluginPaths(staticPluginDir string, pluginDir string) ([]string, error) {
+// GenerateSeparatePluginPaths takes the staticPluginDir and pluginDir and returns separate lists of plugin paths.
+func GenerateSeparatePluginPaths(staticPluginDir string, pluginDir string) ([]string, []string, error) {
```

Comment:
We do not need to export it if this is used internally only

```suggestion
func generateSeparatePluginPaths(staticPluginDir, pluginDir string) ([]string, []string, error) {
```